StatBot/StatBot/Config.cs

```diff
@@ -14,10 +14,8 @@ public class Config
 
         [JsonProperty("discord_token")]
         public string DiscordToken { get; set; }
-
-        /// <summary>
-        /// The prefix the discord bot will use to recognize as a command.
-        /// </summary>
+        
+        /// The prefix the discord bot will use to recognize a command.        
```

Comment:
does this work? I.e. when you hover over the `CommandPrefix` property in your editor, does your IDE show you this summary? I was under the impression that the XML summary tags were required.

Reply:
It is required. Forgot about the IDE functionality of the summary tags